src/lib/viewers/BaseViewer.js

```diff
@@ -575,8 +575,8 @@ class BaseViewer extends EventEmitter {
 
         if (this.annotator && this.areNewAnnotationsEnabled()) {
             this.annotator.emit(ANNOTATOR_EVENT.setVisibility, false);
-            this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
             this.disableAnnotationControls();
+            this.processAnnotationModeChange(this.annotationControlsFSM.transition(AnnotationInput.RESET));
```

Comment:
Should this be in `disableAnnotationControls`, instead?

Reply:
Yeah, you're right